packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -825,6 +874,118 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 	return <DefaultLineItemSublabel product={ product } />;
 }
 
+export function LineItemBillingInterval( { product }: { product: ResponseCartProduct } ) {
+	const translate = useTranslate();
+
+	if ( isDIFMProduct( product ) ) {
+		return <span>{ translate( 'One-time fee' ) }</span>;
+	}
+
+	if ( product.is_included_for_100yearplan ) {
+		return null;
+	}
+
+	if ( isMonthlyProduct( product ) ) {
+		return <span>{ translate( 'Billed every month' ) }</span>;
+	}
+
+	if ( isYearly( product ) ) {
+		return <span>{ translate( 'Billed every year' ) }</span>;
+	}
+
+	if ( isBiennially( product ) ) {
+		return <>{ translate( 'Billed every two years' ) }</>;
+	}
+
+	if ( isTriennially( product ) ) {
+		return <>{ translate( 'Billed every three years' ) }</>;
+	}
+}
+
+function LineItemMetaInfo( { product }: { product: ResponseCartProduct } ) {
```

Comment:
This new component manages all of the 'additional' info we tend to tack onto line items. We can look at this as just a list of 'non billing interval' related things. Each condition should match a specific product, and all of its additional items stored within.

Reply:
This would be a good comment to add to the code itself 😄